### ISSUE TITLE
Users.find() returns wrong data

### ISSUE THREAD

**Issue by centi** [Letsgood/synergykit-sdk-nodejs#1]
When I try to get all users from my project, the returned data (array) contains only one item - the last user:

```javascript
var user = Skit.User();
user.find({
	success: function(users, statusCode) {
		// `users` contains only the last user
	},
	error: function(error, statusCode) {
	}
});
```

I traced the error to the `lib\synergykit.js` modules `request` method, in the part where it handles an Array type of the response. It overwrites the data for every iteration of the array. I think this part should look something like this:

```javascript
if(result.length > 0) {
	response = result
	object.set('data', response)
}else{
	//...
}
```

**Comment by Neklan** [comment]
Hello centi,

this bug is in the next update of SDK fixed.

**Comment by centi** [comment]
Cool, thanks for the info. Do you have some estimate on when the new SDK update will be released?

**Comment by Neklan** [comment]
Solved in version 1.2.1
